v2/mfa/backend/second-factor.mdx

```diff
@@ -657,7 +793,86 @@ Passwordless.init({
 TODO
 </TabItem>
 <TabItem value="python">
-TODO
+
+```python
+from supertokens_python.recipe.passwordless.interfaces import APIInterface, APIOptions
+from typing import Union, Dict, Any, Optional
+from supertokens_python.recipe.session.asyncio import get_session
+from supertokens_python.recipe.usermetadata.asyncio import get_user_metadata
+from supertokens_python.recipe.passwordless.asyncio import get_user_by_id
+
+
+def override_passwordless_apis(original_implementation: APIInterface):
+    original_consume_code_post = original_implementation.consume_code_post
+    original_create_code_post = original_implementation.create_code_post
+
+    async def create_code_post(
```

Comment:
You should highlight this whole function. Since the length of `consume_code_post` in this snippet is also longish

Reply:
done.